CondCore/PPSTimingCalibrationPlugins/interface/PPSTimingCalibrationPayloadInspectorHelper.h

```diff
@@ -0,0 +1,141 @@
+#ifndef CONDCORE_PPSTIMINGCALIBRATIONPLUGINS_PPSTIMINGCALIBRATIONPAYLOADINSPECTORHELPER_H
+#define CONDCORE_PPSTIMINGCALIBRATIONPLUGINS_PPSTIMINGCALIBRATIONPAYLOADINSPECTORHELPER_H
+
+// User includes
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "CondCore/Utilities/interface/PayloadInspectorModule.h"
+#include "CondCore/Utilities/interface/PayloadInspector.h"
+#include "CondCore/CondDB/interface/Time.h"
+#include "CondFormats/PPSObjects/interface/PPSTimingCalibration.h"
+
+// system includes
+#include <fmt/printf.h>
+#include <memory>
+#include <sstream>
+
+// ROOT includes
+#include "TCanvas.h"
+#include "TStyle.h"
+#include "TH2F.h"
+#include "TLatex.h"
+#include "TGraph.h"
+
+namespace PPSTimingCalibrationPI {
```

Comment:
Namespaces should start with the lowercase character, as specified in https://cms-sw.github.io/cms_coding_rules.html